content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -22,9 +23,13 @@ Boards supported by the Science Journal App require special firmware to connect.
 
 4. Click on **Examples** in the left-most sidebar to open the Examples tab.
 
-   - If you're using Nano 33 BLE Sense, search for "Nano33BLESenseFirmware" and open the  the result in the ARDUINO_SCIENCEJOURNAL library.
+   - If you're using the Nano RP2040 Connect, search for "Arduino_ScienceKitCarrier" and open the "ScienceJournal" example in the ARDUINO_SCIENCEKITCARRIER library.
 
-     ![Opening the Nano33BLESenseFirmware example from the Arduino_ScienceJournal library.](img/web-editor-sj-firmware-sj-open.png)
+      ![Opening the ScienceJournal example from the Arduino_ScienceKitCarrier library.](img/SJ_Firmware_rp2040.png)
+
+   - If you're using Nano 33 BLE Sense, search for "Nano33BLESenseFirmware" and open the result in the ARDUINO_SCIENCEJOURNAL library.
+
+      ![Opening the Nano33BLESenseFirmware example from the Arduino_ScienceJournal library.](img/web-editor-sj-firmware-sj-open.png)
 
    - MKR WiFi 1010: Search for `PhysicsLabFirmware` and open the result from the PHYSICSLABFIRMWARE library, **not** the ARDUINO_SCIENCEJOURNAL library.
```

Comment:
This is wrong, when using the MKR WiFi 1010, they need to use the sketch in the Arduino_ScienceJournal library